security/middleware.py

```diff
@@ -126,46 +119,33 @@ def _render_throttling(self, request, exception):
         return get_callable(settings.THROTTLING_FAILURE_VIEW)(request, exception)
 
     def process_view(self, request, callback, callback_args, callback_kwargs):
-        connection = get_connection()
-
-        if getattr(connection, 'input_logged_request', False):
+        if getattr(request, 'input_logged_request', False):
```

Comment:
další chybka, logged_request by se měl brát z requestu ne z connection působí to problémy v testech při rollbacku (v connection logged_request zůstane i po rollbacku a další test je ovlivněný tím prvním)